fluent/fluent.go

```diff
@@ -418,6 +419,13 @@ func (f *Fluent) connect(ctx context.Context) (err error) {
 		f.conn, err = f.dialer.DialContext(ctx,
 			f.Config.FluentNetwork,
 			f.Config.FluentHost+":"+strconv.Itoa(f.Config.FluentPort))
+	case "tls":
+		config := &tls.Config{InsecureSkipVerify: true}
```

Comment:
This fixed value can't be enabled for everyone.
Could you add a config parameter to specify `InsecureSkipVerify` and set `false` in default?

Reply:
And the variable name `config` is too generic. Could you use `tlsConfig` or something else here?

Reply:
Done

Reply:
@zonito Oh, no, I meant that:
* we should have a parameter in [Config](https://github.com/fluent/fluent-logger-golang/blob/master/fluent/fluent.go#L45-L72) about `InsecureSkipVerify` (named probably as `TlsInsecureSkipVerify`)
* and its default value should be `false`
* then use that value here

Reply:
IMO, we should be able to skip the verification (by specifying `InsecureSkipVerify: true`), but using `true` in default is too dangerous. Users must specify it by themselves by their own intention.